src/chart.zoom.js

```diff
@@ -203,18 +180,17 @@ function doZoom(chartInstance, zoom, center, whichAxes) {
 function panIndexScale(scale, delta, panOptions) {
 	var labels = scale.chart.data.labels;
 	var lastLabelIndex = labels.length - 1;
-	var offsetAmt = Math.max((scale.ticks.length - ((scale.options.gridLines.offsetGridLines) ? 0 : 1)), 1);
-	var panSpeed = panOptions.speed;
 	var minIndex = scale.minIndex;
-	var step = Math.round(scale.width / (offsetAmt * panSpeed));
-	var maxIndex;
+	var maxIndex = scale.maxIndex;
 
-	zoomNS.panCumulativeDelta += delta;
+	var dIndex = scale.getValueForPixel(Math.abs(delta)) - scale.getValueForPixel(0);
```

Comment:
Is `dIndex` short for `deltaIndex`? If so, I'd prefer that as a variable name to be a bit clearer